src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -6073,7 +6069,7 @@ private void VisitTupleDeconstructionArguments(ArrayBuilder<DeconstructionVariab
                     int valueSlot;
                     if (underlyingConversion.IsIdentity)
                     {
-                        if ((variable.Expression as BoundLocal)?.DeclarationKind == BoundLocalDeclarationKind.WithInferredType)
+                        if (((variable.Expression as BoundLocal)?.DeclarationKind == BoundLocalDeclarationKind.WithInferredType))
```

Comment:
>if (((variable.Expression as BoundLocal)?.DeclarationKind == BoundLocalDeclarationKind.WithInferredType)) [](start = 24, length = 105)

Is this change necessary?

Reply:
No, the intent was to put it back as it was, I missed a paren.

---
In reply to: [329735618](https://github.com/dotnet/roslyn/pull/38769#discussion_r329735618) [](ancestors = 329735618)